java/pom.xml

```diff
@@ -371,7 +371,6 @@
                         <phase>validate</phase>
                         <configuration>
                             <tasks>
-                                <mkdir dir="${native.build.path}"/>
```

Comment:
When I perform a clean build with this change in the conda environment I get:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:1.3:run (cmake) on project cudf: An Ant BuildException has occured: The directory /home/jlowe/src/cudf/java/target/cmake-build does not exist -> [Help 1]
```

Reply:
to avoid ambiguities what is the exact command you tried?

Reply:
Nothing fancy,  just `mvn clean test`:
```
$ mvn clean test
[INFO] Scanning for projects...
[INFO] 
[INFO] ---------------------------< ai.rapids:cudf >---------------------------
[INFO] Building cudfjni 0.19-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:3.1.0:clean (default-clean) @ cudf ---
[INFO] Deleting /home/jlowe/src/nvcudf/java/target
[INFO] 
[INFO] --- maven-antrun-plugin:1.3:run (cmake) @ cudf ---
[INFO] Executing tasks
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.418 s
[INFO] Finished at: 2021-02-16T16:51:37-06:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:1.3:run (cmake) on project cudf: An Ant BuildException has occured: The directory /home/jlowe/src/nvcudf/java/target/cmake-build does not exist -> [Help 1]
```

Reply:
I can't repro this. What is your cmake version?

Reply:
ah reading the later comment your cmake is `3.18.4`. I tested with `3.18.5`.